packages/orchestrator-ui-components/src/components/WfoTree/treeUtils.ts

```diff
@@ -32,3 +41,38 @@ export const sortTreeBlockByLabel = (
     }
     return 0;
 };
+
+const countChildeNodes = (node: TreeBlock): number => {
+    let count = 0;
+    for (const child of node.children) {
+        count += 1 + countChildeNodes(child);
+    }
+    return count;
+};
+
+export const getPositionInTree = (
```

Comment:
Since this function is a bit complex, maybe add some explanation above

Reply:
Done